project-addons/sale_point_programme/sale_participations_cron.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="points_and_participations_cron" model="ir.cron">
```

Comment:
Este fichero no hace falta ya lo tenéis en project-addons/sale_point_programme/views/sale_participations_cron.xml

Reply:
borrado